### PR TITLE
Update storage and avatars services to include methods to get URLs

### PR DESCRIPTION
**Pull request by stnguyen90**
Issue: https://github.com/appwrite/sdk-for-flutter/issues/71

SDK Generator PR: https://github.com/appwrite/sdk-generator/pull/495

### CODE REVIEW

**Comment by stnguyen90** [comment]
@lohanidamodar, you can focus on https://github.com/appwrite/sdk-for-flutter/pull/74/commits/6a75c0ce80b9876df622800889c7e5d874018884 to see the changes to the storage and avatars services.

File: lib/services/storage.dart [hunk 1]
```diff
@@ -88,156 +83,175 @@ class Storage extends Service {
...
+            borderWidth: borderWidth,
+            borderColor: borderColor,
+            borderRadius: borderRadius,
+            opacity: opacity,
+            rotation: rotation,
+            background: background,
+            output: output,
+        );
+
+        final res = await client.call(HttpMethod.get, path: uri.path[1:], params: uri.queryParameters, responseType: ResponseType.bytes);
```
Comment (stnguyen90) [inline lib/services/storage.dart:227]: Oops! `uri.path[1:]` is python syntax. We could do something like:

```suggestion
        final path = "/" + uri.pathSegments.sublist(1).join("/");

        final res = await client.call(HttpMethod.get, path: path, params: uri.queryParameters, responseType: ResponseType.bytes);
```

**Comment by Shiba-Kar** [comment]
Done This 
```yml 
appwrite:
    git:
      url: https://github.com/appwrite/sdk-for-flutter.git
      ref: feat-71-method-generate-file-url
```
getting this error 
```
('../../../../../AppData/Local/Pub/Cache/git/sdk-for-flutter-9a770bc75904727660acc7b58a1a9344e8ae68dc/lib/src/client.dart').
[        ] Try correcting the name to the name of an existing getter, or defining a getter or field named 'endpoint'.
[        ]         final uri = Uri.parse(client.endpoint + path).replace(queryParameters: params);
```
We want to get the image Url its a must have feature.

**Comment by stnguyen90** [comment]
> We want to get the image Url its a must have feature.

@Shiba-Kar, this is a pretty stale branch and only a POC. I would suggest building the URL manually instead of trying to use this branch.

**Pull request closed without merging**